### PR TITLE
OPSEXP-1257 make sure alpine base is updated

### PR DESCRIPTION
**Pull request by gionn**
OPSEXP-1257

image on dockerhub is not updated on regular basis, e.g.:

```
3.15
Last pushed 3 months ago by [doijanky](https://hub.docker.com/u/doijanky)
```

### CODE REVIEW

**Review by mteodori — CHANGES_REQUESTED** [review]
please replace with 3.15.0 without sha256 that seems incorrect, semver is enough, no need to update packages

**Comment by mteodori** [comment]
> * in addition to that, if you set `3.15.0` you are not getting any bugfix update unless **manually** updating to `3.15.1` when it became available.

this is fine, dependabot can create the PR when there is semver

File: Dockerfile [hunk 1]
```diff
@@ -70,7 +70,9 @@ ENV JAVA_HOME=/usr/lib/jvm/java-${JAVA_MAJOR}-openjdk
 ENV LANG C.UTF-8
 ENV LC_ALL C.UTF-8
 
-RUN apk add openjdk${JAVA_MAJOR}-${JDIST}-headless
+RUN apk update && \
```
Comment (mteodori) [inline Dockerfile:73]: we can use the short form `apk -U upgrade` and add `--no-cache` to all commands or add `rm -vrf /var/cache/apk/*` after to save a few MB of package index - see: https://newbedev.com/alpine-dockerfile-advantages-of-no-cache-vs-rm-var-cache-apk

**Review by mteodori — APPROVED** [review]
minor change

**Review by alxgomz — APPROVED** [review]
LGTM

**Pull request merged**